### PR TITLE
Changed the project info page

### PR DESCRIPTION
**Pull request by FaizanKhan7**
This PR solves issue #225 

**What does this PR do?**
### Shows some info which is shown below in screenshots

**Any background context you want to provide?**

**Screenshots?**
* mobile-view(320px)
mobile-view(320px)

* tab-view(768px)
laptop-view(1024px)

* laptop - 1024px
tab-view(768px)

### CODE REVIEW

File: pages/project/[pid].jsx [hunk 1]
```diff
@@ -31,6 +34,50 @@ const project = () => {
   return (
     <div>
       <Header />
+      <div className={styles.projectProfile}>
+        <div className={styles['card-left-column']}>
+          <h1>Open Source Code</h1>
```
Comment (vipulsnp) [inline pages/project/[pid].jsx:39]: This data should be different for each repo.

File: pages/project/[pid].jsx [hunk 2]
```diff
@@ -31,6 +34,50 @@ const project = () => {
...
+            </div>
+            <p>
+              There should be some content her. So I am filling this with random
+              content inorder to fill this space. Feel free to add on.We can add
+              more and more content here so that we can see what this is gonna
+              look like on the real website page
+            </p>
+            <div className={styles['git-dev-icons']}>
+              <div className={styles['github-icon']}>
+                <img src="/icons/github-icon.png" alt="Github-icon" />
```
Comment (vipulsnp) [inline pages/project/[pid].jsx:54]: Change it to a button and add `link` to the github repository.

File: pages/project/[pid].jsx [hunk 3]
```diff
@@ -31,6 +34,50 @@ const project = () => {
...
+                <p>Github</p>
+              </div>
+              <div className={styles['gitpod-icon']}>
+                <img src="/icons/gitPod-icon.png" alt="Gitpod-icon" />
```
Comment (vipulsnp) [inline pages/project/[pid].jsx:58]: Add gitpod link here and change to button

File: pages/project/[pid].jsx [hunk 4]
```diff
@@ -31,6 +34,50 @@ const project = () => {
   return (
     <div>
       <Header />
+      <div className={styles.projectProfile}>
+        <div className={styles['card-left-column']}>
+          <h1>Open Source Code</h1>
+          <div className={styles['card-left-info']}>
+            <div className={styles['org-languages']}>
+              <p>By Organisation | 08 May 2020</p>
```
Comment (vipulsnp) [inline pages/project/[pid].jsx:42]: `Date` and `Organisation name` Should be from repo data

File: pages/project/[pid].jsx [hunk 5]
```diff
@@ -31,6 +34,50 @@ const project = () => {
...
+              <span className={styles['org-lang-1']}>Javascript</span>
+              <span className={styles['org-lang-2']}>CSS</span>
+            </div>
+            <p>
+              There should be some content her. So I am filling this with random
```
Comment (vipulsnp) [inline pages/project/[pid].jsx:47]: `Description` should be from from repo data

File: pages/project/[pid].jsx [hunk 6]
```diff
@@ -31,6 +34,50 @@ const project = () => {
   return (
     <div>
       <Header />
+      <div className={styles.projectProfile}>
+        <div className={styles['card-left-column']}>
+          <h1>Open Source Code</h1>
+          <div className={styles['card-left-info']}>
+            <div className={styles['org-languages']}>
+              <p>By Organisation | 08 May 2020</p>
+              <span className={styles['org-lang-1']}>Javascript</span>
```
Comment (vipulsnp) [inline pages/project/[pid].jsx:43]: the `language` tags should be from the repo data.

**Review by vipulsnp — CHANGES_REQUESTED** [review]

**Review by vipulsnp — CHANGES_REQUESTED** [review]
@FaizanKhan7 
1. Add `description` .
2. 
image Not properly aligned `forks` `stars` and text
3. 
image
make the size of buttons smaller on Mobile view
4. Align the name of repo to center on Mobile view

**Comment by FaizanKhan7** [comment]
## Resolved Requested changes 
image
image

**Review by vipulsnp — APPROVED** [review]

**Pull request merged**